src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace.cs

```diff
@@ -142,15 +143,36 @@ protected override void Dispose(bool finalize)
                 document.CloseTextView();
             }
 
-            var exceptions = ExportProvider.GetExportedValue<TestExtensionErrorHandler>().GetExceptions();
+            var exceptions = Flatten(ExportProvider.GetExportedValue<TestExtensionErrorHandler>().GetExceptions());
```

Comment:
I'm trying to do the inverse.  I'm trying to unwrap all the exceptions into the final list.  i may have many differnet exceptions, some of which are AggregateExceptions or not.  But i could just reuse their helpers here.  So i'll do that.